erratatool.yml

```diff
@@ -143,6 +144,29 @@ boilerplates:
 
       All OpenShift Container Platform 4.14 users are advised to upgrade to these updated packages and images when they are available in the appropriate release channel. To check for available updates, use the OpenShift Console or the CLI oc command. Instructions for upgrading a cluster are available at https://docs.openshift.com/container-platform/4.14/updating/updating-cluster-cli.html
     solution: *common_solution
+  microshift:
+    synopsis: MicroShift RPM
+    topic: |
+      Red Hat build of MicroShift release 4.14.z is now available with updates to packages and images that fix several bugs.
+    description: |
+      Red Hat build of MicroShift is Red Hat's light-weight Kubernetes orchestration solution designed for edge device deployments and is built from the edge capabilities of Red Hat OpenShift. MicroShift is an application that is deployed on top of Red Hat Enterprise Linux devices at the edge, providing an efficient way to operate single-node clusters in these low-resource environments.
+
+      This advisory contains the RPM packages for Red Hat build of MicroShift 4.14.z. See the following advisory for the container images for this release:
+
+      <ADVISORY_URL> e.g. https://access.redhat.com/errata/RHSA-2023:1234
+
+      All of the bug fixes may not be documented in this advisory. See the following release notes documentation for details about these changes:
+
+      https://access.redhat.com/documentation/en-us/microshift/4.14/html/release_notes/index
+
+
+      All Red Hat build of MicroShift 4.14 users are advised to use these updated packages and images when they are available in the RPM repository.
+    solution: |
+      MicroShift 4.[y].[z] - RPMs
+
+      For MicroShift 4.[y], read the following documentation, which will be updated shortly for this release, for important instructions on how to install the latest RPMs and fully apply this asynchronous errata update:
```

Comment:
```suggestion
      For the Red Hat build of MicroShift 4.[y], read the following documentation, which will be updated shortly for this release, for important instructions on how to install the latest RPMs and fully apply this asynchronous errata update:
```

Reply:
I know this seems obvious, but wanted to check. You want this change in 4.13 and 4.14, correct? @ShaunaDiaz